### PR TITLE
Fix less variable overrides by executing less-modify.js between the build and saving

### LINKED ISSUE #443: Less variable overrides are not always applied during CSS build
@vangheem I'm puzzled. When I set a LESS variable through resources registry control panel, in the development mode it's properly applied for all bundles. When I build the bundles (TTW) for production mode, variables are applied at least for the plone-bundle, but not my barceloneta-derived bundle.

I looked into LESS->CSS-compilation in resourceregistry-pattern and it looked quite simple: all bundle builds get the same less-variables.js before less-compilation.

Any idea, what could be so much different in barcelona.plone.less when compared to plone.less so that variable-overrides are applied in development mode, but not in build?

### PR DESCRIPTION
**Pull request by datakurre**
See also #443
[fixes #443]

### CODE REVIEW

**Comment by jensens** [comment]
seems this needs a rebase with master. also @bloodbare can you look at this?

**Comment by datakurre** [comment]
I rebased, but don't have a current Plone 5 project to test this with. Yet, the issue should be easily confirmed as in #443.

File: mockup/patterns/resourceregistry/js/builder.js [hunk 1]
```diff
@@ -150,6 +154,16 @@ define([
               self.finished(true);
             }
           });
+        }else if($styles.length === config.less.length){
+          $styles.each(function(){$(this).remove();});
+
+          script = document.createElement('script');
+          script.setAttribute('type', 'text/javascript');
+          script.setAttribute('src', self.rview.options.data.lessModifyUrl);
```
Comment (vangheem) [inline mockup/patterns/resourceregistry/js/builder.js]: I looked at this code and it just seemed funny. At this point, wouldn't self.rview.options.data.lessModifyUrl be null/undefined? So why are we loading it this way?

**Comment by vangheem** [comment]
I read the two tickets and I understand what is going on now. Thank you for catching this.

**Pull request merged**